packages/model-viewer/src/three-components/SmoothControls.ts

```diff
@@ -605,31 +608,37 @@ export class SmoothControls extends EventDispatcher {
 
   private onTouchStart = (event: TouchEvent) => {
     this.onPointerDown(() => {
-      if (event.touches.length === 1) {
-        self.addEventListener('touchend', this.onTouchEnd);
+      const {targetTouches, changedTouches, touches} = event;
+      if (targetTouches.length === changedTouches.length) {
```

Comment:
Determines the start of a new interaction, irrespective of the number of touches.
In the case of two touches happening at the same time, Android fires two separate `touchstart` events, each having `targetTouches.length`, `changedTouches.length` equal to `1`.
On the other hand, iOS fires one `touchstart` event. `targetTouches.length` and `changedTouches.length` should both be equal to `2`.